lib/unix_socket.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+var http    = require('http'),
+    debug   = require('./debug'),
+    CONFIG  = require('../config/config');
+
+var exports = module.exports = {};
+
+exports.send = function send(data, cb) {
+  try {
+    var options = {
+      socketPath: CONFIG.SOCKET_PATH,
+      path: CONFIG.SOCKET_URL,
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/x-protobuf'
+      }
+    };
+
+    var callback = function callback(res) {
+      if (res.statusCode == 200) {
+        return cb({'success': true, 'statusCode': 200});
+      } else {
+        // If status_code is not 200, attempt to retry every 5 min. delay
+        setTimeout(function () {
+          exports.send(data, cb);
+        }, CONFIG.DELAY.FIVE_MINUTES_DELAY);
```

Comment:
Lets set the delay to 10 seconds with a max retries of 6.

Reply:
ok, i'll update this functionality.